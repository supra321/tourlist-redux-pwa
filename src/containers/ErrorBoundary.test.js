import React from 'react';
import {shallow} from 'enzyme';
import ErrorBoundary from './ErrorBoundary';

it('Expect to render ErrorBoundary Component',()=>{
	expect(shallow(<ErrorBoundary />)).toMatchSnapshot();
})

it('Sets the ErrorState to true',()=>{
	const wrapper=shallow(<ErrorBoundary />);
	/*wrapper.find('[props]').simulate('event');*/
	expect(wrapper.state()).toEqual({ErrorState:false});
	/*expect(wrapper.props().propName).toEqual({propValue});*/
})
