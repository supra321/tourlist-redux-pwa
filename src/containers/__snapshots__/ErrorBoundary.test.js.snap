// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render ErrorBoundary Component 1`] = `ShallowWrapper {}`;
